ros2pkg/ros2pkg/verb/prefix.py

```diff
@@ -24,10 +25,16 @@ def add_arguments(self, parser, cli_name):
         arg = parser.add_argument(
             'package_name',
             help='The package name')
+        parser.add_argument(
+            '--share',
+            action='store_true',
+            help='show share directory for the package')
         arg.completer = package_name_completer
 
     def main(self, *, args):
         prefix_path = get_prefix_path(args.package_name)
         if prefix_path is None:
             return 'Package not found'
         print(prefix_path)
+        if args.share:
+            print('Share dir: %s' % get_share_directory(args.package_name))
```

Comment:
In the current form this doesn't seem very useful beyond being read by a user.

Instead I would suggest to only show the share directory when using the new option. And also not show any text in front of the path. That will enable to use the result in cases where you want to construct a path, e.g. `cat $(ros2 pkg prefix --share PKGNAME)/package.xml`.

Reply:
Please take a look at the latest version